motoman_ar2010_support/config/default_kinematic_offsets.yaml

```diff
@@ -0,0 +1,43 @@
+kinematic_offsets:
+  1_s:
+    x: 0
+    y: 0
+    z: 0
+    roll: 0
+    pitch: 0
+    yaw: 0
```

Comment:
The config files for forge_descriptions uses a different format for the origin. Does it make sense to use the same format in both places?

Reply:
I kind of wrestled with that, I'm not really sure.
They aren't technically related at all so they don't need to be the same, but it would make sense that if one wins out it is used in both places.
 I ended up keeping this format since it was what we were using before for kinematic offsets, but I prefer the format in forge_descriptions. I'll take another look at it.